H4LCppModule.py

```diff
@@ -88,6 +88,7 @@ def beginFile(self, inputFile, outputFile, inputTree, wrappedOutputTree):
         self.out.branch("pTZ2",  "F")
         self.out.branch("etaZ2",  "F")
         self.out.branch("phiZ2",  "F")
+        self.out.branch("massZ2_2j",  "F")
```

Comment:
You need to add all kinematic information for _2j case. 

Currently, its only mass. You need to add rapidity, energy, phi, etc.